src/commands/runDraftTool/runDraftIngress.ts

```diff
@@ -0,0 +1,749 @@
+import {CompletedSteps} from './model/guidedExperience';
+import * as vscode from 'vscode';
+import {Context} from './model/context';
+import {StateApi, State} from '../../utils/state';
+import {longRunning} from '../../utils/host';
+import {downloadDraftBinary, runDraftCommand} from './helper/runDraftHelper';
+import {
+   AzureWizard,
+   AzureWizardExecuteStep,
+   AzureWizardPromptStep,
+   IActionContext
+} from '@microsoft/vscode-azext-utils';
+import {
+   Az,
+   AzApi,
+   CertificateItem,
+   DnsZoneItem,
+   getAzCreds,
+   KeyVaultItem,
+   ManagedClusterItem,
+   ResourceGroupItem,
+   SubscriptionItem
+} from '../../utils/az';
+import {Errorable, failed, getAysncResult} from '../../utils/errorable';
+import {sort} from '../../utils/sort';
+import {
+   ignoreFocusOut,
+   PromptResourceGroup,
+   PromptSubscription
+} from './helper/commonPrompts';
+import {getAsyncOptions} from '../../utils/quickPick';
+import {ValidateRfc1123} from '../../utils/validation';
+import {parseAzureResourceId} from '@microsoft/vscode-azext-azureutils';
+import {
+   KnownCertificatePermissions,
+   KnownSecretPermissions
+} from '@azure/arm-keyvault';
+import {buildUpdateIngressCommand} from './helper/draftCommandBuilder';
+
+const INGRESS_FILENAME = 'ingress.yaml';
```

Comment:
We cannot. This comes from Draft and is how Draft names all the variables. In the future, we can change how Draft works but for now we are stuck with this. It would also be nice to get this from a "dry run" rather than hard code it in the future.